### PR TITLE
Fix for missing recordCount display

### PR DESCRIPTION
**Pull request by Augen2203**
Design edits, and fix for missing recordCount display when state is true, in bootstrap5 theme
Снимок экрана 2021-12-04 в 02 04 08

### CODE REVIEW

**Comment by Augen2203** [comment]
Similarly, for example, instead of getting an unrelated translation for each of the "`recordCount`" parts, we can make it an integral part, something like:
```
<small class="text-muted d-block">
                    {{ __('Displayed records: :from-:to of :total',[
                        'from' => ($paginator->currentPage() -1 ) * $paginator->perPage() + 1,
                        'to' => ($paginator->currentPage() -1 ) * $paginator->perPage() + count($paginator->items()),
                        'total' => $paginator->total(),
                    ]) }}
                </small>
```

How is it implemented in that orchid platform

**Comment by luanfreitasdev** [comment]
Hi @Augen2203 , 

thanks for sending the PR!, it will help a lot

File: resources/views/components/frameworks/bootstrap5/pagination.blade.php [hunk 1]
```diff
@@ -1,48 +1,73 @@
...
+        <span class="font-semibold">{{ $paginator->total() }}</span>
+        {{ trans('livewire-powergrid::datatable.pagination.results') }}
+    </small>
+@elseif($recordCount === 'short')
+    <small class="text-muted d-block">
+        <span class="font-semibold"> {{ $paginator->firstItem() }}</span>
+        -
+        <span class="font-semibold"> {{ $paginator->lastItem() }}</span>
+        |
+        <span class="font-semibold"> {{ $data->links()->paginator->total() }}</span>
```
Comment (luanfreitasdev) [inline resources/views/components/frameworks/bootstrap5/pagination.blade.php:18]: $data variable is not available here

```suggestion
        <span class="font-semibold"> {{ $paginator->total() }}</span>
```

**Review by luanfreitasdev — CHANGES_REQUESTED** [review]
good work

**Review by luanfreitasdev — APPROVED** [review]

**Pull request merged**